java/arcs/core/storage/StorageProxy.kt

```diff
@@ -467,12 +503,19 @@ class StorageProxy<Data : CrdtData, Op : CrdtOperationAtTime, T>(
     }
 
     private fun notifyUpdate(data: T) {
+        // If this isn't our first update and the data's hashCode is equivalent to the old data's
+        // hashCode, no need to send an update.
+        if (updateNotifications > 0 && lastDataUpdateHash == data?.hashCode()) return
+        updateNotifications++
```

Comment:
Is there any need for a count? Could this just be a boolean flag like `firstUpdateIsSent`?